kubeportal/tests/test_api.py

```diff
@@ -160,6 +167,29 @@ def test_logout(self):
         response = self.post(f'/api/{API_VERSION}/logout/')
         self.assertEqual(response.status_code, 200)
 
+    def test_option_preflight_without_auth(self):
+        self.jwt = None
+        self.csrf = None
+
+        test_path = ['/api/',
+                     f'/api/{API_VERSION}/users/{self.admin.pk}',
+                     f'/api/{API_VERSION}/users/{self.admin.pk}/webapps',
+                     f'/api/{API_VERSION}/groups/{self.admin_group.pk}',
+                     f'/api/{API_VERSION}/cluster/k8s_apiserver',
+                     f'/api/{API_VERSION}/login/'
+        ]
+        for path in test_path:
+            if 'login' in path:
```

Comment:
Woah, ugly. What about putting the request method in the list above?

Reply:
yeah it is the first time you describe my code as ugly. Great. thanks. :)